docs/UserGuide.md

```diff
@@ -581,7 +581,7 @@ was executed.
 
 ---
 
-## 6. FAQ
+## 6. FAQ (Aizat)
```

Comment:
Feels like for FAQ maybe we can say its done by all? To show teamwork in some sense, just a suggestion.

Reply:
Are we allowed to? I seem to have a vague memory of Jane saying it has to be one member one section.

Reply:
Okay in that case, it should be fine then.